src/MultiTable.test.js

```diff
@@ -17,7 +17,7 @@ it('can change tables browser', () => {
   let browserSelect = wrapper.find('.browserSelect');
 
   // Change the table
-  browserSelect.simulate('change', {value: 'Patient'});
+  browserSelect.simulate('change', {target: {value: 'Patient'}, test: true});
```

Comment:
Why is the `test` parameter here needed?

Reply:
It was an artifact from a previous way I was doing the testing. Updated